doc/quick-start-user-guide.md

```diff
@@ -0,0 +1,20 @@
+# Quick Start User Guide
+
+This guide demonstrates how to consume the python bindings for GlusterD-2.0 APIs.
+We create a two-node GlusterFS cluster using these bindings.
+
+## Setup
+This guide takes the following as an example of IPs for the two nodes:
+ * **Node 1**: `192.168.56.101`
+ * **Node 2**: `192.168.56.102`
+ 
```

Comment:
I think you have to make this more comprehensive. You specify two nodes here but in the example you have a single node. It is pretty confusing.

Reply:
done.